diagnostic.md

```diff
@@ -35,7 +37,9 @@ Record your responses inside the fenced code blocks below each question.
     ```
 
     ```md
-    <!-- your response here -->
+    1. In the first example, the 'product_id' route is nested under the 'products' route function as opposed to the second example which is not a nested route but has an extended URL path.
+
+    2. The first URL example does not contain the nested path (i.e. the 'products' part) and the second example contains a path that isnt defined as a function and has an unnecissarily long URL (i.e. the 'products' part.)
```

Comment:
👍 

More explanation:
The first `'product'` route definition is nested under the `'products'`
definition, whereas the second `'product'` route definition is a sibling of
the `'products'` definition.

Nested route definitions, like the first `'product'` route above, require
a nested file structure, such that Route and Template files for `'product'`
are expected to be found at `app/products/product`. For sibling routes,
folders are siblings as well.

The first `'product'` route definition instructs Ember to render
`app/products/product/template.hbs` in the context (the `outlet`) of the
parent `app/products/template.hbs`. The second `'product'` route definition
instructs Ember to render `app/product/template.hbs` in the context of the
application template.

These two route definitions do **not** have different paths.